src/Effects.js

```diff
@@ -1445,6 +1445,40 @@ module.exports = class Effects {
         }
       }
     };
+
+    this.squiggles = {
+      points: [],
+      dotSpacing: 3,
+      amplitude: 40,
+      period: 400,
+      dotRadius: 7,
+      numSquiggles: 8,
+      init: function () {
+        this.points = [];
+        p5.noStroke();
+        p5.angleMode(p5.DEGREES);
+        let numPoints = p5.height / this.dotSpacing;
+        for (var i = 0; i < numPoints; i++) {
+          this.points.push({
+            x: 0,
```

Comment:
Do all points start at the bottom of the page?

Reply:
`point.x` is set each tick of the draw loop, so setting to 0 here just to initialize it to something.